docs/userstories/add_imagestream_to_jh.md

```diff
@@ -0,0 +1,21 @@
+```gherkin
+Feature: Add new ImageStream to JupyterHub Spawner UI
+    Given I am a user of MOC-CNV
+    And I have access to JupyteHub on MOC-CNV
+    And there exists an image quay.io/thoth-station/s2i-minimal-notebook
+    When I update templates/imagestream-template.yaml with:
+        metadata.annotations = {
+            opendatahub.io/notebook-image-url: "https://github.com/thoth-station/s2i-minimal-notebook"
+            opendatahub.io/notebook-image-name: "Test minimal notebook Image"
+            opendatahub.io/notebook-image-desc: "Jupyter notebook image with minimal dependency set to start experimenting with Jupyter environment."
+
+        }
+        metadata.name = test-s2i-minimal-notebook
+        spec.tags[0].annotations = {
+            openshift.io/imported-from: quay.io/thoth-station/s2i-minimal-notebook
+        }
+        spec.tags[0].from.name = quay.io/thoth-station/s2i-minimal-notebook:v0.0.4
```

Comment:
Do you want to add another line as:
`spec.tags[0].from.name = v0.0.4` to update the imagestream tag as well?

Reply:
done